utils_test.go

```diff
@@ -66,13 +66,13 @@ func TestConsulClientReturnErrorOnBadResponse(t *testing.T) {
 	config.Registry.FailLimit = 1
 	config.Registry.FailWaitTime = 0
 
-	consul, err := getConsulClient(config)
+	consul, err := GetConsulClient("test-service-name", "test-profile", config)
 	if consul != nil || err == nil {
 		t.Error("Error should be raised")
 	}
 
-	if err.Error() != "Bad response from Consul service" {
-		t.Error("Wrong error message")
+	if err.Error() != "Cannot get connection to consul" {
```

Comment:
why this is changed? We have separate test for this called 'TestConsulClientReturnErrorOnTimeout'

Reply:
I have rewrote the consul functions, so I made some change for test.

Reply:
case with bad response still in code https://github.com/edgexfoundry/device-sdk-go/pull/50/files#diff-7d1c2a3334601b6c1958aae0a594cba8R130 and we have no test for it. Moreover currently we have two tests which check same case with 'Cannot get connection to consul'.